### PR TITLE
Add troubleshooting info for installing missing pyscard dependency

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
Looks like this will have to wait for the new main branch before being merged.

### CODE REVIEW

**Comment by robin-nitrokey** [comment]
We should also mention or link this [here](https://github.com/Nitrokey/nitrokey-documentation/blob/main/source/components/software/nitropy/all-platforms/installation.rst).

**Review by robin-nitrokey — APPROVED** [review]

File: nitrokey3/troubleshooting.rst.inc [hunk 1]
```diff
@@ -109,3 +109,21 @@ Currently the web updater doesn't support the Nitrokey 3. However you can get th
 
 
 .. _nfc.fail: https://nfc.fail/
+
+Piv Troubleshooting
```
Comment (jans23) [inline nitrokey3/troubleshooting.rst.inc:113]: "PIV"

**Comment by jans23** [comment]
Re restructured our documentation. Please change this PR to branch "main".

**Comment by jans23** [comment]
I see it's replaced by #306.

**Pull request closed without merging**